json-guides/circuit-breaker.json

```diff
@@ -303,10 +303,10 @@
               "Now that you learned about circuit breakers and fallbacks, you can explore the parameters in the <code>@CircuitBreaker</code> annotation and see the circuit in action.",
               "<br>You learned about the following parameters:",
               "<ul>",
-              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made before determining if the circuit should trip. This also specifies the size of the rolling window. The default is <code>20</code> requests.",
+              "<li><b>requestVolumeThreshold</b>: The minimum number of requests that must be made to the service before determining whether the circuit trips. This parameter also specifies the size of the rolling window. The default is <code>20</code> requests.",
               "<li><b>failureRatio</b>: The minimum failure ratio in the rolling window to trigger the circuit breaker. The default ratio is <code>0.5</code>.",
-              "<li><b>delay</b>: The delay in milliseconds to transition to a half-open state after the circuit opens. The default is <code>5000</code> ms.",
-              "<li><b>successThreshold</b>: The number of consecutive successful invocations of the service that is required before the circuit closes. The default is <code>1</code> request.",
+              "<li><b>delay</b>: The delay in milliseconds after the circuit opens to transition to a half-open state. The default is <code>5000</code> ms.",
```

Comment:
The `delay` definition should also be changed in the step we introduce it.

Reply:
Changed